tsdb/chunkenc/float_histogram.go

```diff
@@ -954,6 +954,12 @@ func (it *floatHistogramIterator) Next() ValueType {
 			it.err = err
 			return ValNone
 		}
+
+		if !histogram.IsValidSchema(schema) {
+			it.err = fmt.Errorf("invalid histogram schema %d", schema)
```

Comment:
As above, can we re-use `ErrHistogramsInvalidSchema`?

Reply:
done all in 5b39b79f5a41a25c70a9efd842ca21e55c888649